prac_07/myguitars.py

```diff
@@ -0,0 +1,20 @@
+from prac_06.guitar import Guitar
+import csv
```

Comment:
I dont think import csv is being used yet, unless you were planning to finish this task